c2cgeoportal/scaffolds/create/mapserver/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM camptocamp/mapserver:latest
+MAINTAINER Camptocamp "info@camptocamp.com"
+
+ADD include/*.map /etc/mapserver/include/
```

Comment:
The bsae image I'm using want the file to be named locally mapserver.map because of this line:
ONBUILD COPY mapserver.map /etc/mapserver/

That's the reason why the makefile is copying c2cgeoportal.map to mapserver.map before creating the image.